src/Overlay.js

```diff
@@ -1,13 +1,26 @@
 /*eslint-disable object-shorthand, react/prop-types */
```

Comment:
Ah.. Then I would re-write it similarly to: [src/ProgressBar.js#L1](https://github.com/react-bootstrap/react-bootstrap/blob/0683df74ce3510ef7d525b7e4dd01274178400c5/src/ProgressBar.js#L1)
as
```js
/* eslint react/prop-types: [2, {ignore: ["container", "containerPadding", "target", "placement", "children"] }] */
/* These properties are validated in 'Portal' and 'Position' components */
```

--
again: I can do it myself later, if you wish